workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  workflow_call:
+    inputs:
+      test-grammar:
+        type: boolean
+        default: true
+      test-rust:
+        type: boolean
+        default: false
+
+jobs:
+  grammar:
+    name: Test grammar
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-14]
+    if: ${{inputs.test-node}}
```

Comment:
It says `test-grammar` above. Either are fine to me, but pick one :)

Reply:
Oh, whoops.